hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/db.json

```diff
@@ -378,11 +378,11 @@
           }
         ],
         "missingSince": 1578491371528,
-        "pipelineId": "05e3d908-ff01-4ce6-ad75-f3ec79bcc7982"
+        "pipelineID": "05e3d908-ff01-4ce6-ad75-f3ec79bcc7982"
       },
       {
-        "id": 2,
-        "keys": 5943,
+        "containerID": 2,
```

Comment:
Not understood why these are changed, and how it is related to this change.

Reply:
@bharatviswa504 These are not related to the change. But, I noticed that the json structure of missing containers api mock response does not match with what is expected by the UI and thought of fixing them along with this change. Since, these are mock responses, they will not affect any behavior of Recon, except when starting a local mock development server to serve mock responses from this db.json file.